pkg/log/wrapper.go

```diff
@@ -46,37 +47,32 @@ type Logger struct {
 	level Level // Level specifies a level of verbosity for V logs.
 }
 
-// LogDepth logs a message with the given level and depth.
-func (l *Logger) LogDepth(calldepth int, level Level, msg string, args ...any) {
-	l.log.LogDepth(calldepth+1, level, msg, args...)
-}
-
 // Log logs a message with the given level.
 func (l *Logger) Log(level Level, msg string, args ...any) {
-	l.LogDepth(0, level, msg, args...)
+	l.log.Log(context.TODO(), level, msg, args...)
 }
 
 // Debug logs a debug message.
 func (l *Logger) Debug(msg string, args ...any) {
-	l.LogDepth(0, DebugLevel, msg, args...)
+	l.log.Log(context.TODO(), DebugLevel, msg, args...)
 }
 
 // Info logs an informational message.
 func (l *Logger) Info(msg string, args ...any) {
-	l.LogDepth(0, InfoLevel, msg, args...)
+	l.log.Log(context.TODO(), InfoLevel, msg, args...)
 }
 
 // Warn logs a warning message.
 func (l *Logger) Warn(msg string, args ...any) {
-	l.LogDepth(0, WarnLevel, msg, args...)
+	l.log.Log(context.TODO(), WarnLevel, msg, args...)
 }
 
 // Error logs an error message.
 func (l *Logger) Error(msg string, err error, args ...any) {
 	if err != nil {
-		args = append(args[:len(args):len(args)], slog.Any(slog.ErrorKey, err))
+		args = append(args[:len(args):len(args)], slog.Any("err", err))
```

Comment:
It seems that the logic of the `slog` has big changed

Reply:
Yeah it certainly has a decent amount of changes. Hopefully when it gets added to stdlib things will slow down :)